### ISSUE TITLE
negative values of lr-means column

### ISSUE THREAD

**Issue by ngvananh2508** [ventolab/CellphoneDB#192]
Hi, I used celphonedb to infer cell cell communication and I got some negative values of both lr-means, receptor-means and ligand_means. Does my data have issues or if not, how can I interpret these negative values
![image](https://github.com/ventolab/CellphoneDB/assets/130348585/0a6a2bc8-7279-4964-930f-b1e29ba2f7eb)
![image](https://github.com/ventolab/CellphoneDB/assets/130348585/4b5074b3-78e8-4f85-893e-9e5c1d31bdf1)

Thank you very much for helping me!

**Comment by ktroule** [comment]
Hi.

It seems that might have scaled the data which might transform some counts to negative values. 
Please, normalize your data without transforming to negative values.

**Comment by ngvananh2508** [comment]
No, my read-count matrix was scran-normalized, it does not contain any negative values of the entries.
